src/commands/lint/install.rs

```diff
@@ -0,0 +1,210 @@
+                "version `{}` does not exist. Available versions: {:?}",
+                version,
+                available_versions
+            );
+        }
+        // Return the required toolchain version and the name of the linter tag or `main` that
+        // corresponds to the desired version.
+        (lookup_toolchain_version(version)?, version)
+    }
+    // No version was passed in the `InstallArgs` open a dialog with all available versions
+    // (including the main branch) to choose from.
+    else {
+        let Some(selection) = dialoguer::FuzzySelect::new()
+            .with_prompt("Available `bevy_lint` versions")
+            .items(&available_versions)
+            .interact_opt()?
+        else {
+            debug!("no version selected");
+            return Ok(());
+        };
+
+        let version = &available_versions[selection];
+        debug!("selected {}", version);
+
+        let required_toolchain = lookup_toolchain_version(version)?;
+
+        // If no specific version was passed, ask for confirmation.
+        if !dialoguer::Confirm::new()
+            .with_prompt(format!(
+                "Do you want to install `bevy_lint-{version}` and the required toolchain: `{}` ?",
+                required_toolchain.toolchain.channel
+            ))
+            .interact()
+            .context(
+                "failed to show interactive prompt, try passing a specific version as an argument",
+            )?
+        {
+            anyhow::bail!(
+                "User does not want to install `bevy_lint-{version}` and the required toolchain: `{}`",
+                required_toolchain.toolchain.channel
+            );
+        }
+
+        // Return the required toolchain version and the name of the linter tag or `main` that
+        // corresponds to the desired version.
+        (required_toolchain, version)
+    };
+
+    install_toolchain(&rust_toolchain)?;
+
+    // Prepare command to install `bevy_lint`
+    let mut cmd = CommandExt::new("rustup");
+
+    cmd.arg("run")
+        .arg(rust_toolchain.toolchain.channel)
+        .arg(env::var_os("BEVY_CLI_CARGO").unwrap_or("cargo".into()))
+        .arg("install")
+        .arg("--git")
+        .arg(GIT_URL);
+
+    if version == "main" {
+        cmd.arg("--branch").arg("main");
+    } else {
+        cmd.arg("--tag").arg(format!("lint-{version}"));
+    }
+
+    cmd.arg("--locked")
+        .arg("bevy_lint")
+        .ensure_status(AutoInstall::Never)
+        .context(format!("failed to install `bevy_lint-{version}`"))?;
+
+    Ok(())
+}
+
+/// Print the available `bevy_lint` versions, including main, in a table to stdout.
+pub(crate) fn list() -> anyhow::Result<()> {
+    let releases = list_available_releases()?;
+
+    let mut table = comfy_table::Table::new();
+
+    table
+        .load_preset(comfy_table::presets::UTF8_FULL)
+        .apply_modifier(comfy_table::modifiers::UTF8_ROUND_CORNERS)
+        .set_header(vec!["Bevy Lint Version"]);
```

Comment:
```suggestion
        .set_header(["Bevy Lint Version"]);
```

(Nit) Turns out arrays also implement `Into<Row>`, so we can use that instead of `Vec`